browser/react-wallet/scratch.js

```diff
@@ -0,0 +1,13 @@
+const trinsic = require('@trinsic/trinsic');
+
+let service = new trinsic.AccountService();
+let details = new trinsic.AccountDetails();
+details.setName("Michael Black");
+details.setEmail("michaelblack117@gmail.com");
```

Comment:
you might want a bogus email here
```suggestion
details.setEmail("info@trinsic.id");
```

Reply:
Oh shoot yeah I need to delete that file altogether as I was just using that to test some stuff early on